charts/neon-pg-sni-router/README.md

```diff
@@ -34,13 +34,15 @@ Kubernetes: `^1.18.x-x`
 | exposedService.annotations | object | `{}` | Annotations to add to the exposed service |
 | exposedService.httpsPort | int | `nil` | Exposed Service https port. If null, https server will not be exposed. |
 | exposedService.port | int | `5432` | Exposed Service proxy port |
+| exposedService.portTls | int | `5433` | Exposed Service proxy port that uses TLS to computes |
```

Comment:
Will the 5433 value be specified by cplane as dst port, right?

Reply:
Yes, and it's all configurable both on the pg-sni-router side and on control-plane side